ui/ducks/send/send.js

```diff
@@ -992,8 +992,10 @@ const slice = createSlice({
         recipient.error = null;
         recipient.warning = null;
       } else {
-        const isSendingToken = asset.type === ASSET_TYPES.TOKEN;
         const { chainId, tokens, tokenAddressList } = action.payload;
+        const assetAddress =
+          asset.details && asset.details.address ? asset.details.address : '';
```

Comment:
Can this be `asset?.details?.address ?? ''`?

Reply:
Yes, it can be. 
Fixed in: 51054012cfbb3e51b2853adafa9e9d8fd155656d